README.md

```diff
@@ -0,0 +1,66 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TDD_ruby
+
+> In this project we implement a class with some methods, but we do it by using test-driven development (TDD). The idea is to write tests first and then the code.
+
+Additional description about the project and its features.
+
+## Built With
+
+- Ruby
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+```
+Ruby
+```
+
+### Setup
+```
+git clone git@github.com:isaka-lumato/tdd_ruby.git
+cd tdd_ruby.git
+```

```

Comment:
- [ ] It's a small change but it is very important for you to provide all the commands about your project for the other users. In the README file, Please mention the step on how to test the project. [HInt: `rspec`]